site/assets/js/snippets.js

```diff
@@ -76,9 +76,9 @@
   }
 
   // Indeterminate checkbox example
-  document.querySelectorAll('.bd-example-indeterminate [type="checkbox"]')
-    .forEach(checkbox => {
-      checkbox.indeterminate = true
+  document.querySelectorAll('.bd-example-indeterminate')
```

Comment:
Proposed in BS PR another way around. Let's merge yours here and see what's the preferred way in BS.